Configurations/ConfigCommon.xcconfig

```diff
@@ -20,9 +20,11 @@ SPARKLE_ICON_NAME = AppIcon
 SPARKLE_VERSION_MAJOR = 2
 SPARKLE_VERSION_MINOR = 0
 SPARKLE_VERSION_PATCH = 0
-SPARKLE_VERSION = $(SPARKLE_VERSION_MAJOR).$(SPARKLE_VERSION_MINOR).$(SPARKLE_VERSION_PATCH)
+SPARKLE_VERSION_SUFFIX =
+SPARKLE_VERSION = $(SPARKLE_VERSION_MAJOR).$(SPARKLE_VERSION_MINOR).$(SPARKLE_VERSION_PATCH)$(SPARKLE_VERSION_SUFFIX)
 
 CURRENT_PROJECT_VERSION = $(SPARKLE_VERSION)
```

Comment:
You do want to update this to some hardcoded counter we can manually increase right? Maybe we can start it with `2000`..

Reply:
I've set it to `2000`. If it were my own project, I'd want to automatically increment whatever the committed/current value of this build setting is when tagging a release, but you've explained your reluctance toward this and I understand and am happy to leave it here.